src/main/java/org/fogbowcloud/saps/engine/core/dispatcher/restlet/resource/EmailResource.java

```diff
@@ -0,0 +1,73 @@
+package org.fogbowcloud.saps.engine.core.dispatcher.restlet.resource;
+
+import org.apache.commons.httpclient.HttpStatus;
+import org.apache.log4j.Logger;
+import org.fogbowcloud.saps.engine.core.archiver.storage.PermanentStorage;
+import org.fogbowcloud.saps.engine.core.archiver.storage.PermanentStorageType;
+import org.fogbowcloud.saps.engine.core.archiver.storage.exceptions.InvalidPropertyException;
+import org.fogbowcloud.saps.engine.core.archiver.storage.exceptions.PermanentStorageException;
+import org.fogbowcloud.saps.engine.core.archiver.storage.nfs.NfsPermanentStorage;
+import org.fogbowcloud.saps.engine.core.archiver.storage.swift.SwiftPermanentStorage;
+import org.fogbowcloud.saps.engine.core.dispatcher.email.TasksEmailBuilder;
+import org.fogbowcloud.saps.engine.utils.SapsPropertiesConstants;
+import org.restlet.data.Form;
+import org.restlet.data.MediaType;
+import org.restlet.representation.Representation;
+import org.restlet.representation.StringRepresentation;
+import org.restlet.resource.Post;
+import org.restlet.resource.ResourceException;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Properties;
+
+public class EmailResource extends BaseResource {
+
+    private static final Logger LOGGER = Logger.getLogger(EmailResource.class);
+
+    //TODO update value this to "tasks_id[]" (requires a change to the Dashboard component)
+    private static final String REQUEST_ATTR_PROCESSED_TASKS = "images_id[]";
+
+    @Post
+    public Representation sendTaskToEmail(Representation representation) {
+        Form form = new Form(representation);
+
+        String userEmail = form.getFirstValue(UserResource.REQUEST_ATTR_USER_EMAIL, true);
+        String userPass = form.getFirstValue(UserResource.REQUEST_ATTR_USERPASS, true);
+
+        if (!authenticateUser(userEmail, userPass) || userEmail.equals("anonymous"))
+            throw new ResourceException(HttpStatus.SC_UNAUTHORIZED);
+
+        String[] tasksId = form.getValuesArray(REQUEST_ATTR_PROCESSED_TASKS, true);
+        Properties properties = application.getProperties();
+
+        try {
+            PermanentStorage permanentStorage = createPermanentStorage(properties);
```

Comment:
This is something I am not sure about, maybe I can load the properties in static and final fields, and just use them. As this will not be tested, I added a TODO to check if it is possible to reuse it instead of creating another one every time.